src/main/java/com/github/manolo8/darkbot/Main.java

```diff
@@ -103,9 +113,14 @@ public Main() {
             Popups.showMessageAsync("Error", I18n.get("bot.issue.config_load_failed"), JOptionPane.ERROR_MESSAGE);
 
         API.createWindow();
+        initializeStartupSettings(params);
```

Comment:
```suggestion
        if (params.shouldAutoStart()) setRunning(true);
```
This being a single line used in a single place, i'd just avoid the function